packages/ui/src/components/va-file-upload/VaFileUploadGalleryItem/VaFileUploadGalleryItem.vue

```diff
@@ -76,11 +76,11 @@ export default defineComponent({
       backgroundColor: colorToRgba(props.color, 0.7),
     }))
 
-    const classesComputed = computed(() => ({
-      'va-file-upload-gallery-item--not-image': !previewImage.value,
-      'va-file-upload-gallery-item--focused': isFocused.value,
-      'va-file-upload-gallery-item--undo': removed.value,
-    }))
+    const classesComputed = useBem('va-file-upload-gallery-item', {
+      notImage: !previewImage.value,
+      focused: isFocused.value,
+      undo: removed.value,
+    })
```

Comment:
```suggestion
    const classesComputed = useBem('va-file-upload-gallery-item', () => ({
      notImage: !previewImage.value,
      focused: isFocused.value,
      undo: removed.value,
    }))
```
To have reactivity we need to use a callback function or a reactive object.
And the same below